.circleci/config.yml

```diff
@@ -109,12 +205,46 @@ jobs:
 workflows:
   smart_contract:
     jobs:
-      - run_tests:
+      - setup:
+          context: Development
+      - swap_tests:
+          context: Development
+          requires:
+            - setup
+      - indexer_tests:
+          context: Development
+          requires:
+            - setup
+      - index_tests:
+          context: Development
+          requires:
+            - setup
+      - delegate_tests:
+          context: Development
+          requires:
+            - setup
+      - delegate_factory_tests:
+          context: Development
+          requires:
+            - setup
+      - types_tests:
           context: Development
+          requires:
+            - setup
+      - wrapper_tests:
+          context: Development
+          requires:
+            - setup
       - deploy:
           context: NPM_Publish
           requires:
-            - run_tests
+            - swap_tests
```

Comment:
when do the js tests run?

Reply:
@poiresel look at line 65

Reply:
*63

Reply:
@poiresel which js tests? the ones that are not related to solidity?